File: aoj/volume100/python3/A10020.py
import sys

counts = {}
for i in range(ord('a'), ord('z')+1):
    counts[chr(i)] = 0

for line in sys.stdin:
    for char in line.lower():
        if char in counts:
            counts[char] += 1

for i in range(ord('a'), ord('z')+1):
    char = chr(i)
    print("%s : %d" % (char, counts[char]))

File: aoj/volume100/scala/A10020.scala
object Main {
  def main(args: Array[String]) {
    var counts = scala.collection.mutable.Map[Char, Int]()
    for {
      char <- 'a' to 'z'
    } counts(char) = 0

    val lines = Iterator.continually(scala.io.StdIn.readLine()).takeWhile(_ != null)
    for {
      line <- lines
      char <- line.toLowerCase
      if char.isLower
    } counts(char) += 1

    for {
      char <- 'a' to 'z'
    } println(s"$char : ${counts(char)}")
  }
}
